code/__HELPERS/string_lists.dm

```diff
@@ -12,3 +12,20 @@ GLOBAL_LIST_EMPTY(string_lists)
 		return
 
 	return GLOB.string_lists[string_id] = values
+
+///A wrapper for baseturf string lists, for support for non list values, and a stack_trace if we have major issues
+/datum/proc/baseturfs_string_list(list/values, turf/T)
+	if(!islist(values))
+		values = list(values)
```

Comment:
I'm not seeing any situations in the replaced code where a non-list value would get converted into a list.

Reply:
There's shit in assemble_baseturfs(). but it's an assumption in baseturfs code, and I don't want to have to deal with that.